projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -897,6 +902,44 @@ export class MapOlService {
     }
   }
 
+  private create_wfs_layer(l: VectorLayer): olVectorLayer {
+
+    // Update URL with current projection
+    const url = getUrlFromUri(l.url);
+    const paras = getParameterJsonFromUri(l.url);
+    // The below re-projection of the bbox is not required, because in WFS (contrary to WMS)
+    // a bbox ('BBOX') may have another projection than the output-projection ('SRSNAME')
+    // if (paras['SRSNAME'] !== this.EPSG) {
+    //   if (paras['BBOX']) {
+    //     this.reprojectFeatures()
+    //   }
+    // }
+    paras['SRSNAME'] = this.EPSG;
+    let fullUrl = `${url}?`;
+    for (const para in paras) {
```

Comment:
To create url params you could also use `URLSearchParams` like e.g. in [updateSearchParamsHashRouting](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/v7.3.0-next.2/projects/demo-maps/src/app/route-components/route-example-events/route-map3.component.ts#L133)

Reply:
done.